### ISSUE TITLE
s=input() can't go thru

### ISSUE THREAD

**Issue by y11z** [spyder-ide/spyder#17626]
### What steps will reproduce the problem?

s=input() can't go thru

was all fine until this upgrade! how can i work around or downgrade?

### Traceback
```python-traceback
Traceback (most recent call last):
  File "C:\ProgramData\Anaconda3\lib\site-packages\qtconsole\base_frontend_mixin.py", line 138, in _dispatch
    handler(msg)
  File "C:\ProgramData\Anaconda3\lib\site-packages\spyder\plugins\ipythonconsole\widgets\debugging.py", line 278, in _handle_input_request
    return super(DebuggingWidget, self)._handle_input_request(msg)
  File "C:\ProgramData\Anaconda3\lib\site-packages\qtconsole\frontend_widget.py", line 512, in _handle_input_request
    self._readline(msg['content']['prompt'], callback=callback, password=msg['content']['password'])
  File "C:\ProgramData\Anaconda3\lib\site-packages\qtconsole\console_widget.py", line 2422, in _readline
    self._show_prompt(prompt, newline=False, separator=False)
TypeError: _show_prompt() got an unexpected keyword argument 'separator'
```

## Versions

* Spyder version: 5.1.5 
* Python version: 3.8.8
* Qt version: 5.12.10
* PyQt5 version: 5.12.3
* Operating System: Windows 10

### Dependencies

```

# Mandatory:
atomicwrites >=1.2.0          :  1.4.0 (OK)
chardet >=2.0.0               :  4.0.0 (OK)
cloudpickle >=0.5.0           :  2.0.0 (OK)
cookiecutter >=1.6.0          :  1.7.3 (OK)
diff_match_patch >=20181111   :  20200713 (OK)
intervaltree >=3.0.2          :  3.1.0 (OK)
IPython >=7.6.0               :  8.2.0 (OK)
jedi >=0.17.2;<0.19.0         :  0.18.1 (OK)
jsonschema >=3.2.0            :  3.2.0 (OK)
keyring >=17.0.0              :  23.4.0 (OK)
nbconvert >=4.0               :  6.1.0 (OK)
numpydoc >=0.6.0              :  1.2 (OK)
paramiko >=2.4.0              :  2.8.1 (OK)
parso >=0.7.0;<0.9.0          :  0.8.3 (OK)
pexpect >=4.4.0               :  4.8.0 (OK)
pickleshare >=0.4             :  0.7.5 (OK)
psutil >=5.3                  :  5.8.0 (OK)
pygments >=2.0                :  2.11.2 (OK)
pylint >=2.5.0;<2.10.0        :  2.9.6 (OK)
pyls_spyder >=0.4.0           :  0.4.0 (OK)
pylsp >=1.2.2;<1.3.0          :  1.2.4 (OK)
pylsp_black >=1.0.0           :  None (OK)
qdarkstyle =3.0.2             :  3.0.2 (OK)
qstylizer >=0.1.10            :  0.1.10 (OK)
qtawesome >=1.0.2             :  1.0.3 (OK)
qtconsole >=5.1.0             :  5.3.0 (OK)
qtpy >=1.5.0                  :  2.0.1 (OK)
rtree >=0.9.7                 :  0.9.7 (OK)
setuptools >=49.6.0           :  58.0.4 (OK)
sphinx >=0.6.6                :  4.4.0 (OK)
spyder_kernels >=2.1.1;<2.2.0 :  2.1.3 (OK)
textdistance >=4.2.0          :  4.2.1 (OK)
three_merge >=0.1.1           :  0.1.1 (OK)
watchdog >=0.10.3             :  2.1.6 (OK)
zmq >=17                      :  22.3.0 (OK)

# Optional:
cython >=0.21                 :  None (OK)
matplotlib >=2.0.0            :  3.4.3 (OK)
numpy >=1.7                   :  1.21.5 (OK)
pandas >=1.1.1                :  1.3.4 (OK)
scipy >=0.17.0                :  1.7.1 (OK)
sympy >=0.7.3                 :  None (OK)
```

**Comment by ccordoba12** [comment]
Hey @y11z, thanks for reporting. This error is fixed in our latest version (5.3.0).

Since it's still not available in Anaconda, you have two options:

1. Use our Windows installer, which you can find [here](https://github.com/spyder-ide/spyder/releases).
2. Create a new environment with conda-forge packages. For that, please close Spyder, open the Anaconda Prompt and run the following commands there:

        conda create -n spyder-cf -c conda-forge spyder
        conda activate spyder-cf
        spyder

**Comment by y11z** [comment]
It all functions. but leave long errors.
thanks
ERROR:asyncio:Exception in callback AddThreadSelectorEventLoop._handle_select([4404], [])handle: <Handle AddThreadSelectorEventLoop._handle_select([4404], [])>Traceback (most recent call last):  File "C:\ProgramData\Anaconda3\envs\spyder-cf\lib\asyncio\events.py", line 80, in _run    self._context.run(self._callback, *self._args)  File "C:\ProgramData\Anaconda3\envs\spyder-cf\lib\site-packages\tornado\platform\asyncio.py", line 579, in _handle_select    self._handle_event(r, self._readers)  File "C:\ProgramData\Anaconda3\envs\spyder-cf\lib\site-packages\tornado\platform\asyncio.py", line 591, in _handle_event    callback()  File "C:\ProgramData\Anaconda3\envs\spyder-cf\lib\site-packages\tornado\platform\asyncio.py", line 189, in _handle_events    handler_func(fileobj, events)  File "C:\ProgramData\Anaconda3\envs\spyder-cf\lib\site-packages\zmq\eventloop\zmqstream.py", line 445, in _handle_events    zmq_events = self.socket.EVENTS  File "C:\ProgramData\Anaconda3\envs\spyder-cf\lib\site-packages\zmq\sugar\attrsettr.py", line 51, in __getattr__    return self._get_attr_opt(upper_key, opt)  File "C:\ProgramData\Anaconda3\envs\spyder-cf\lib\site-packages\zmq\sugar\attrsettr.py", line 63, in _get_attr_opt    return self.get(opt)  File "C:\ProgramData\Anaconda3\envs\spyder-cf\lib\site-packages\zmq\_future.py", line 215, in get    result = super(_AsyncSocket, self).get(key)  File "zmq/backend/cython/socket.pyx", line 464, in zmq.backend.cython.socket.Socket.get  File "zmq/backend/cython/socket.pyx", line 135, in zmq.backend.cython.socket._check_closedzmq.error.ZMQError: not a socket



    On Wednesday, April 6, 2022, 10:45:07 AM EDT, Carlos Cordoba ***@***.***> wrote:  
 
 


Hey @y11z, thanks for reporting. This error is fixed in our latest version (5.3.0).

Since it's still not available in Anaconda, you have two options:
   
   -    
Use our Windows installer, which you can find here.

   -    
Create a new environment with conda-forge packages. For that, please close Spyder, open the Anaconda Prompt and run the following commands there:
    conda create -n spyder-cf -c conda-forge spyder
 conda activate spyder-cf
 spyder
